.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: android
+dist: trusty
```

Comment:
What's trusty?

Reply:
The default android travis-ci file includes this. It seems to specify the build environment. Here's the documentation page on it.
https://docs.travis-ci.com/user/reference/trusty/